extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -682,13 +682,43 @@ export class ConsentUI {
         (this.restrictFullscreenOn_ &&
           this.document_.activeElement !== this.ui_)
       ) {
-        // TODO (@torch2424) Send response back if enter fullscreen was succesful
+        const error = 'Could not enter fullscreen';
+        dev().warn(TAG, error);
+        this.sendIframeMessage_('Error', error);
```

Comment:
Should there be two different errors sent to the iframe: 1 for if the frame isn't visible & another for if the iframe has not received user interaction first? @zhouyx

Reply:
That would be great. Or you could use one message that covers both, for example  "Could not enter fullscreen, fullscreen is only supported when the iframe is visible and after user interaction" : )